<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-P27CLIA" failures="1" tests="4" name="Default test(failed)" time="0.131" errors="0" timestamp="2022-08-09T15:41:37 IST">
  <testcase classname="Practice1.Demo1" name="createContact" time="0.019">
    <skipped/>
  </testcase> <!-- createContact -->
  <testcase classname="Practice1.Demo1" name="createContact" time="0.003">
    <skipped/>
  </testcase> <!-- createContact -->
  <testcase classname="Practice1.Demo1" name="createContact" time="0.003">
    <skipped/>
  </testcase> <!-- createContact -->
  <testcase classname="Practice1.Demo1" name="createContact" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Practice1.Demo1.createContact(Demo1.java:10)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- createContact -->
  <testcase classname="Practice1.Demo1" name="modifyContact" time="0.0">
    <ignored/>
  </testcase> <!-- modifyContact -->
</testsuite> <!-- Default test(failed) -->
